pkg/libmxnet.pkg

```diff
@@ -4,9 +4,9 @@ OPTS = dict(
     name = VAR.fullname,
     description = '''\
 Lightweight, portable deep learning library
- This package provides the core library exposing the MXNet C API.
- .
- It has been built with OpenCV and OpenMP support disabled.
+This package provides the core library exposing the MXNet C API.
+.
```

Comment:
It seems to be a convention in deb-package descriptions, try `apt show [whatever]` to see what I mean.

If I had to guess, I'd say it is presumably so that there are no actual blank lines in the description, while allowing effective paragraphs.